src/main/java/edu/harvard/iq/dataverse/Dataverse.java

```diff
@@ -105,7 +106,7 @@ public void setDataverseType(DataverseType dataverseType) {
     private final String uncategorizedString = "Uncategorized";
 
     /**
-     * @todo Don't hard code these as English.
+     * @todo Don't hard code these as English. Use bundleUtils
```

Comment:
@rbhatta99 It looks like you can remove this todo if you add bundle references to the "Laboratory" and "Research Group" text in this same function you reference the bundle for "Department" in line 128.